src/stores/quizStore.js

```diff
@@ -3,7 +3,10 @@ import { defineStore } from "pinia";
 export const useQuizStore = defineStore("quiz", {
   state: () => ({
     url: "",
-    givenAnswers: [],
+    givenAnswers: {
+      elapsedTime: 180,
```

Comment:
is this the placeholder for now, that will be POSTed?
I know it works, but I cant find where you set this data for the POST request.

Reply:
it is in the created hook. in the body of the post request